js/app.jsx

```diff
@@ -850,16 +908,10 @@ var Settings = React.createClass({
 
 var routes = (
   <Route name="app" path="/" handler={App}>
-    <DefaultRoute handler={SearchableCaseverList}/>
-    <Route name="caseversions"       path="/caseversion" handler={SearchableCaseverList}/>
-    <Route name="caseversion_search" path="/caseversion/search/:query" handler={SearchableCaseverList}/>
-    <Redirect                        from="/search/:query"  to="/caseversion/search/:query" />
-    <Route name="suites_noid"        path="/suite/" handler={SearchableSuiteList}/>
-    <Redirect                        from="/suite"  to="/suite/" />
-    <Route name="suite_search"       path="/suite/search/:query" handler={SearchableSuiteList}/>
-    <Route name="suite"              path="/suite/:id" handler={AddToSuite} />
-    <Route name="settings"           path="/settings" handler={Settings} />
-    <NotFoundRoute handler={SearchableCaseverList}/>
+    <DefaultRoute handler={CaseVerWrapper}/>
+    <Route name="case" path="/case" handler={CaseVerWrapper}/>
```

Comment:
A "Case" is a different concept from "CaseVersion" in MozTrap, please use "caseversion" here.